src/client/sandbox/event/unload.ts

```diff
@@ -5,23 +5,44 @@ import { isFirefox, isIOS } from '../../utils/browser';
 import { overrideDescriptor } from '../../utils/overriding';
 import Listeners from './listeners';
 
+interface EventProperties {
+    storedReturnValue: string;
+    prevented:         boolean;
+    storedHandler:     any;
+    nativeEventName:   string;
+    eventName:         string;
+    eventPropSetter:   any;
+};
+
 export default class UnloadSandbox extends SandboxBase {
     BEFORE_UNLOAD_EVENT: string = 'hammerhead|event|before-unload';
     BEFORE_BEFORE_UNLOAD_EVENT: string = 'hammerhead|event|before-before-unload';
     UNLOAD_EVENT: string = 'hammerhead|event|unload';
+    NATIVE_UNLOAD_EVENT: string = 'unload';
```

Comment:
It is unnecessary to save the event name, see https://github.com/DevExpress/testcafe-hammerhead/blob/daf68f22dc795cc7d976e5da629eb3ff63175208/src/client/sandbox/event/unload.ts#L51